browser-test/src/admin/admin_predicates.test.ts

```diff
@@ -609,6 +618,8 @@ test.describe('create and edit predicates', () => {
     await adminQuestions.addDateQuestion({questionName: 'date-question'})
     await adminQuestions.addCurrencyQuestion({
       questionName: 'currency-question',
+      questionText: '*currency-question*',
```

Comment:
nitpick (non-blocking): take the `-` out of the `questionText` since generally we use those in admin names and not question titles (aka `questionText`).